src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryServiceTest.kt

```diff
@@ -142,4 +142,72 @@ class CosmasInMemoryServiceTest {
                 .build()
         this.service.createVersion(newVersionRequest, createVersionRecorder)
     }
+
+    private fun addPatchToService(text: String, user: String = "No name", fileId: String = "0",
+                                            time: Long = 0) {
+        val createVersionRecorder: StreamRecorder<CosmasProto.CreatePatchResponse> = StreamRecorder.create()
+        val newVersionRequest = CosmasProto.CreatePatchRequest
+                .newBuilder()
+                .setUser(user)
+                .setFileId(fileId)
+                .setText(text)
+                .setTimeStamp(time)
+                .build()
+        this.service.createPatch(newVersionRequest, createVersionRecorder)
+    }
+
+    private fun checkCorrect(user : String, text : String, time : Long,

```

Comment:
Please use spaces around colons consistently. Here you add a space fro both sides while just a few lines above you ass a space from the right side only. [Kotlin code style](https://kotlinlang.org/docs/reference/coding-conventions.html#horizontal-whitespace) defines reasonable rules about spaces around colons.